internal/install/execution/go_task_recipe_executor.go

```diff
@@ -233,26 +234,31 @@ func varFromPrompt(envConfig recipes.VariableConfig) (string, error) {
 		msg = envConfig.Prompt
 	}
 
-	templates := &promptui.PromptTemplates{
-		Prompt:  "{{ . | bold }} ",
-		Valid:   "{{ . | bold }} ",
-		Invalid: "{{ . | bold }} ",
-		Success: "  - {{ . }} ",
-	}
-
-	prompt := promptui.Prompt{
-		Label:     msg,
-		Templates: templates,
-	}
+	value := ""
+	var prompt survey.Prompt
 
 	if envConfig.Secret {
-		prompt.HideEntered = true
-		prompt.Mask = '*'
+		prompt = &survey.Password{
+			Message: msg,
+		}
+	} else {
+		defaultValue := ""
+
+		if envConfig.Default != "" {
+			defaultValue = envConfig.Default
+		}
+
+		prompt = &survey.Input{
+			Message: msg,
+			Default: defaultValue,
+		}
```

Comment:
Would be interesting to consider using the [suggest options](https://github.com/AlecAivazis/survey#suggestion-options) feature vs what we do now with having the default as a part of the [prompt message](https://github.com/newrelic/open-install-library/blob/main/recipes/newrelic/infrastructure/ohi/apache/debian.yml#L40)

Reply:
That'd involve a change to the schema, so wouldn't happen right away. But might offer a better UX.

Reply:
Yeah, that sounds nice.  Perhaps in another pr?

Reply:
Yup, not needed for this one, but possible future enhancement.